features/step_definitions/files_steps.rb

```diff
@@ -26,7 +41,6 @@
 end
 
 
-Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |file_name, content|
-  expect(uncommitted_files).to eql [file_name]
-  expect(IO.read file_name).to eql content
+Then(/^I (?:still|again) have an uncommitted file with name: "([^"]+)" and content: "([^"]+)"$/) do |name, content|
```

Comment:
Wow, that's pretty hardcore! I'm using [ag.vim](https://github.com/rking/ag.vim) to find all matching locations. After changing the first result, and going to the next result, I use a custom key combination to re-apply the last change and go to the next search result.

```viml
function ReplaceAndGoToNext()
  :normal! .
  :cnext
endfunction
nnoremap <silent> <C-]> :call ReplaceAndGoToNext()<CR>
```